### PR TITLE
port num change

### PR DESCRIPTION
**Pull request by catbelljune**
port num change check plz

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-PORT = 3003
\ No newline at end of file
+PORT = 3002
+# port Num Change
```
Comment (pentagram5) [inline .env:2]: qwe

Comment (pentagram5) [reply .env:2]: qweqwe

**Review by pentagram5 — COMMENTED** [review]
qwe

File: .env (hunk 1 above)
Comment (catbelljune) [reply .env:2]: just change

Comment (pentagram5) [reply .env:2]: ok^^

**Review by pentagram5 — APPROVED** [review]

**Comment by catbelljune** [comment]
thx!

**Pull request merged**